### DISCUSSION TITLE
Lodestar Standup 2024-5 (February 6, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6390] [Standups]
### Meeting Info

[Feb 6, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/feb-6-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - Monthly PR Grooming
  - Holesky Fork Prep
  - v1.15.1 hotfixes
  - v1.16.0 Planning
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by nflaig** [comment]
Would like to quickly discuss how we wanna go about our eslint rules, and what justifies adding a new rule as I don't think necessary more is better in that case as being over restrictive for no good reason can decrease devex in my opinion.

**Comment by nflaig** [comment]
Does anyone have an idea who is managing the buildjet runners? Maybe we can talk about that, looks like a cloud solution that offers external runners, see https://buildjet.com/for-github-actions

Maybe this would be an easy way to stabilize our tests as recently we have seen more timeouts for some reason which might indicate that current runners are overloaded. Does anyone else in Chainsafe use those?

**Comment by philknows** [comment]
Closing for #6418
